style.css

```diff
@@ -0,0 +1,102 @@
+body {
+    background-color: #172b4d;
+}
+.container {
+    margin: 0 auto;
+    width: 315px;
+    height: 4210px;
+    background-color: #fff;
+    background-image: url('img/background.png'); 
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 30px;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Kindly make sure the background image is visible on your section , like it is in the Figma design :+1: 

| ### Your design| ### The figma design |
| :---:| :---:|
| ![Screenshot from 2023-01-13 18-36-52](https://user-images.githubusercontent.com/80458199/212372418-3609b75e-61e4-4b1e-8677-b2c0d8ae5eb9.png) | ![Screenshot from 2023-01-13 18-40-13](https://user-images.githubusercontent.com/80458199/212372639-a1ca2769-1c1b-4902-b5d2-91d324863dbd.png)|